config_template.cfg

```diff
@@ -34,9 +34,6 @@ KIND_VERIFICATION_SOLUTION= NO_VERIFICATION_SOLUTION
 % Mathematical problem (DIRECT, CONTINUOUS_ADJOINT, DISCRETE_ADJOINT)
 MATH_PROBLEM= DIRECT
 %
-% Regime type (COMPRESSIBLE, INCOMPRESSIBLE)
-REGIME_TYPE= COMPRESSIBLE
```

Comment:
Hi, I am too late to the party, but I just noticed that `PHYSICAL_PROBLEM` a few lines above wasn't adapted in the config_template. So maybe that can be sneaked in one the next PR's.